site/docs/v1/tech/apis/tenants.adoc

```diff
@@ -74,6 +75,64 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_tenant-response-body.adoc[]
 
+
+== Create a New Tenant From an Existing Tenant
+
+This API is used to create a new Tenant from an existing Tenant.
```

Comment:
I could go either way on whether we call this out here, but I think it should be consistent.

```suggestion
This API is used to create a new Tenant from an existing Tenant.

Specifying an Id on the URI will instruct FusionAuth to use that Id when creating the Tenant. Otherwise, FusionAuth will generate an Id for the Tenant.
```

Reply:
I dont think that we need it